jest.config.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  collectCoverageFrom: [
+    "packages/*/src/**/*.mjs",
+    "packages/*/src/**/*.js",
+    "codemods/*/src/**/*.mjs",
+    "codemods/*/src/**/*.js",
+    "eslint/*/src/**/*.mjs",
+    "eslint/*/src/**/*.js",
+  ],
+  // The eslint/* packages use ESLint v6, which has dropped support for Node v6.
```

Comment:
Nit: could you add a todo item here that we should remove the process.version check in Babel 8? By that time we will drop Node < 8.9 support.